client/js/components/CardEventSmall/CardEventSmall.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {
+  Text,
+  View,
+  Image,
+  ImageBackground,
+  TouchableOpacity,
+} from 'react-native';
+import {RatingScore00} from '../RatingScore';
```

Comment:
Please do not include numeric characters inside your component or class names.

Reply:
Done

![image](https://user-images.githubusercontent.com/20639360/70753113-ddf47f80-1ce8-11ea-805d-fc0a165b5008.png)